src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -360,15 +360,15 @@ public function eq($position)
      *         return $node->text();
      *     });
      *
-     * @param \Closure $closure An anonymous function
+     * @param callable $callable An anonymous function
      *
      * @return array An array of values returned by the anonymous function
      */
-    public function each(\Closure $closure)
+    public function each(callable $callable)
```

Comment:
I made a callableToClosure helper so it works for now. But all \Closure typehints should be callable in 4.0 ?